pkg/skuba/actions/cluster/init/init.go

```diff
@@ -163,31 +159,41 @@ func writeScaffoldFiles(initConfiguration InitConfiguration) error {
 		return errors.Wrapf(err, "could not change to cluster directory %q", initConfiguration.ClusterName)
 	}
 	for _, file := range scaffoldFilesToWrite {
-		filePath, _ := filepath.Split(file.Location)
-		if filePath != "" {
-			if err := os.MkdirAll(filePath, 0700); err != nil {
-				return errors.Wrapf(err, "could not create directory %q", filePath)
-			}
-		}
-		f, err := os.Create(file.Location)
-		if err != nil {
-			return errors.Wrapf(err, "could not create file %q", file.Location)
-		}
-		str, err := renderTemplate(file.Content, initConfiguration)
-		if err != nil {
-			return errors.Wrap(err, "unable to render template")
-		}
-		_, err = f.WriteString(str)
-		if err != nil {
-			return errors.Wrapf(err, "unable to write template to file %s", f.Name())
+		if err := WriteScaffoldFile(file, initConfiguration); err != nil {
+			return err
```

Comment:
nice refactoring.
nit: "WriteScaffoldFile" doesn't mean anything to me though. If people understand what it means, good. Else I would try to rename it with something clearly explaining what it does, like WriteTemplatedFile.

Reply:
I preserved the name on purpose, because I've seen the concept of `scaffold` here and there when it comes to templates. Maybe this should go into another PR that focuses on the naming of things. That being said, and maybe I'm biased because of [Rails](https://github.com/rails/rails), to me `scaffold` and `template` and pretty much synonyms, and in this context I understand it like `configuration and project files that the tool is going to automatically produce for you when calling X command`. So, in this case, it makes sense.

If other people disagree, as I said, it can go into another PR. But I'd say that this kind of change would be out of place for this PR.